File: AkitaModel/Speciality.cs
namespace Skyware.Lis.AkitaModel;

/// <summary>
/// Medical speciality of a doctor, e.g. General practitioner, Surgeon, etc.
/// </summary>
public class Speciality
{

    /// <summary>
    /// PK in LIS iLab.
    /// </summary>
    public int? Id { get; set; }

    /// <summary>
    /// Name of the speciality.
    /// </summary>
    public string Name { get; set; }

}

File: AkitaModelJava/akitamodel/src/main/java/skyware/lis/akitamodel/Speciality.java
package skyware.lis.akitamodel;

import lombok.Data;

/**
 * Medical speciality of a doctor, e.g., General practitioner, Surgeon, etc.
 */
@Data
public class Speciality {

    /**
     * PK in LIS iLab.
     */
    private Integer id;

    /**
     * Name of the speciality, e.g., "Surgeon".
     */
    private String name;
    
}
